src/main/kotlin/no/nav/syfo/application/api/authentication/JwtIssuer.kt

```diff
@@ -1,12 +1,11 @@
 package no.nav.syfo.application.api.authentication
 
 data class JwtIssuer(
-    val accectedAudienceList: List<String>,
+    val acceptedAudienceList: List<String>,
     val jwtIssuerType: JwtIssuerType,
     val wellKnown: WellKnown
 )
 
 enum class JwtIssuerType {
-    selvbetjening,
-    veileder
+    SYSTEM_V2,
```

Comment:
Kanskje kalle det no ala INTERN_AZUREAD_V2 eller evt. AZUREAD_V2?
Det beskriver kanskje bedre at det er en felles AzureAD-config.

Reply:
Ja, jeg var ikke helt sikker på navnet. Bare AZUREAD_V2 høres bra ut, for det må vel være intern uansett? Eventuelt om man kan lage OBO-flow med sbs-token, så vil dette likevel fungere?

Reply:
OBO-flow med sbs-token vil ikke fungere med denne konfigurasjonen, der er det en annen issuer.